code/modules/tents/deployed_tents.dm

```diff
@@ -55,6 +56,7 @@
 		return
 	var/mob/subject_mob = subject
 	RegisterSignal(subject_mob, list(COMSIG_MOVABLE_TURF_ENTERED, COMSIG_GHOST_MOVED), PROC_REF(mob_moved), override = TRUE) // Must override because we can't know if mob was already inside tent without keeping an awful ref list
+	subject_mob.RegisterSignal(src, COMSIG_PARENT_QDELETING, TYPE_PROC_REF(/mob, tent_deletion_clean_up), override = TRUE)
```

Comment:
why not just RegisterSignal(subject_mob  and avoid having a proc on /mob ?

Reply:
Well we can't, I wish could work that way but there's no signal that works like that currently